nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/EvaluateXPath.java

```diff
@@ -129,8 +129,10 @@ public class EvaluateXPath extends AbstractProcessor {
             .build();
 
     public static final PropertyDescriptor VALIDATE_DTD = new PropertyDescriptor.Builder()
+            .displayName("Allow Document Type Declaration")
             .name("Validate DTD")
-            .description("Specifies whether or not the XML content should be validated against the DTD.")
+            .description("When enabled, Embedded Document Type Declaration will be allowed and it will be used for the XML content validation. "
+                    + "Because of security reasons it is advised to disable this feature.")
```

Comment:
Recommend adjusting the wording to clarify the security implications.
```suggestion
            .description("Allow embedded Document Type Declaration in XML. "
                    + "This feature should be disabled to avoid XML entity expansion vulnerabilities.")
```